plugins/woocommerce-admin/client/products/sections/test/utils.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import { isValidSlug } from '../utils';
+
+describe( 'product section utils', () => {
+	describe( 'isValidSlug', () => {
+		it( 'should return true if a slug is valid', () => {
+			const validity = isValidSlug( 'valid-slug_123' );
+			expect( validity ).toBeTruthy();
```

Comment:
I think `.toBeTrue()` would be better here, since we are in control of `isValidSlug()` and we really should only ever be returing `true` or `false`. It we were to change the implementation (by mistake) of `isValidSlug()` and it returned something like the slug itself, we'd want to catch that. `.toBeTruthy()` would still pass in that case.